tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -223,10 +223,34 @@ def test_with_description(self):
     )
     def test_anomaly_detection_alert(self, mock_seer_request):
         data = {
-            **self.alert_rule_dict,
+            "aggregate": "count()",
+            "query": "",
+            "time_window": 30,
```

Comment:
Is the `time_window` the only other difference between this and `self.alert_rule_dict`? You can just update that rather than define it all again here

Reply:
I also changed the alert thresholds to 0; happy to update instead if you think that's better

Reply:
Oh I see agh the threshold makes it trickier, this is fine